source/deployment/metrics.rst

```diff
@@ -58,177 +59,213 @@ For install instructions, see `Prometheus install guides <https://prometheus.io/
+- ``mattermost_post_file_attachments_total``: The total number of file attachments created because a post was created.
+- ``mattermost_post_pushes_sent_total``: The total number of mobile push notifications sent because a post was created.
+- ``mattermost_post_total``: The total number of posts created.
+- ``mattermost_post_webhooks_totals``: The total number of webhook posts created.
 
 .. image:: ../images/perf_monitoring_messaging_metrics.png
 
-Process Metrics:
+Process Metrics
+^^^^^^^^^^^^^^^
 
-    - ``mattermost_process_cpu_seconds_total``: Total user and system CPU time spent in seconds.
-    - ``mattermost_process_max_fds``: Maximum number of open file descriptors.
-    - ``mattermost_process_open_fds``: Number of open file descriptors.
-    - ``mattermost_process_resident_memory_bytes``: Resident memory size in bytes.
-    - ``mattermost_process_start_time_seconds``: Start time of the process since unix epoch in seconds.
-    - ``mattermost_process_virtual_memory_bytes``: Virtual memory size in bytes.
+- ``mattermost_process_cpu_seconds_total``: Total user and system CPU time spent in seconds.
+- ``mattermost_process_max_fds``: Maximum number of open file descriptors.
+- ``mattermost_process_open_fds``: Number of open file descriptors.
+- ``mattermost_process_resident_memory_bytes``: Resident memory size in bytes.
+- ``mattermost_process_start_time_seconds``: Start time of the process since unix epoch in seconds.
+- ``mattermost_process_virtual_memory_bytes``: Virtual memory size in bytes.
 
-Search Metrics:
+Search Metrics
+^^^^^^^^^^^^^^
 
-    - ``mattermost_search_posts_searches_duration_seconds_sum``: The total duration, in seconds, of search query requests.
-    - ``mattermost_search_posts_searches_duration_seconds_count``: The total number of search query requests.
+- ``mattermost_search_posts_searches_duration_seconds_sum``: The total duration, in seconds, of search query requests.
+- ``mattermost_search_posts_searches_duration_seconds_count``: The total number of search query requests.
 
-WebSocket Metrics:
+WebSocket Metrics
+^^^^^^^^^^^^^^^^^
 
-    - ``mattermost_websocket_broadcasts_total``: The total number of WebSocket broadcasts sent by type.
-    - ``mattermost_websocket_event_total``: The total number of WebSocket events sent by type.
+- ``mattermost_websocket_broadcasts_total``: The total number of WebSocket broadcasts sent by type.
+- ``mattermost_websocket_event_total``: The total number of WebSocket events sent by type.
+    
+Logging Metrics
+^^^^^^^^^^^^^^^
+
+- ``logger_queue_used``: Current logging queue level(s).
+- ``logger_logged_total``: The total number of logging records emitted.
+- ``logger_error_total``: The total number of logging errors.
+- ``logger_dropped_total``: The total number of logging records dropped.
+- ``logger_blocked_total``: The total number of logging records blocked.
     
-Logging Metrics:
+Grafana Debugging Metrics
```

Comment:
I'm not certain this is the "right title", since it sounds like we're debugging Grafana. Maybe just `Debugging Metrics`?

Reply:
Thanks, @lieut-data. I'll get that title updated.